### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by MarcinMoskala** [andrewoma/kotlin-script#10]
I installed as in instruction on Mac:

```
MBP-MarcinMac:scripts marcin$ curl -s https://raw.githubusercontent.com/andrewoma/kotlin-script/master/kotlins > ~/bin/kotlins && chmod u+x ~/bin/kotlins
```

And tried to start simple project: 

```kotlin
#!/usr/bin/env kotlins
println("Hello from Kotlin!")
for (arg in args) {
    println("arg: $arg")
}
```

It is in file named hello.kts. But it is not working correctly:

```
MBP-MarcinMac:scripts marcin$ kotlins hello.kts 
warning: the '-d' option with a directory destination is ignored because '-module' is specified
Error: Could not find or load main class MainKt
```

**Comment by andrewoma** [comment]
What version of kotlin are you using? I haven't tested this against 1.1 yet and there's likely to be breaking changes

**Comment by MarcinMoskala** [comment]
Kotlin version 1.1.0-rc-91 (JRE 1.8.0_91-b14)

**Comment by comm1x** [comment]
I have the same issue. 
But before I find out, that after installation there is no `kotlins` in environments. I moved `kotlins` to `/usr/local/bin/` and add `~/Library/Application Support/AndroidStudio3.0/Kotlin/kotlinc/bin` to PATH, and made `chmod +x kotlin, kotlinc, kotlinc-jvm`. 
And after that I got this issue.

```
warning: the '-d' option with a directory destination is ignored because '-Xbuild-file' is specified
Error: Could not find or load main class MainKt
```

MacOS 10.13.2 High Sierra
Kotlin 1.2.10
